Apps/Medication/src/Delegates/IMedStatementDelegate.cs

```diff
@@ -28,9 +29,9 @@ public interface IMedStatementDelegate
         /// </summary>
         /// <param name="query">The medication statement query execute against the ODR.</param>
         /// <param name="protectiveWord">The protective word to validate.</param>
-        /// <param name="userId">The user querying.</param>
+        /// <param name="hdid">The HDID of the user querying.</param>
         /// <param name="ipAddress">The IP of the user querying.</param>
-        /// <returns>The Medication History response.</returns>
-        Task<MedicationHistoryResponse> GetMedicationStatementsAsync(MedicationHistoryQuery query, string protectiveWord, string userId, string ipAddress);
+        /// <returns>The Medication History response wrapped in an HNMessage.</returns>
+        Task<HNMessage<MedicationHistoryResponse>> GetMedicationStatementsAsync(MedicationHistoryQuery query, string protectiveWord, string hdid, string ipAddress);
```

Comment:
nit: not sure we need HNMessage anymore since that was a wrapper for the Pharmanet communication messages. The MedicationHistoryResponse could be enhanced to have the missing status field